src/application/components/applicationPreviewSubsectionField.tsx

```diff
@@ -1,21 +1,14 @@
 import { Col, Row } from 'react-grid-system';
 import { useTranslation } from 'react-i18next';
 import { connect } from 'react-redux';
-import { RootState } from '../../root/rootReducer';
-import { getFieldTypeMapping } from '../selectors';
 import { FormField } from '../../plotSearch/types';
 import {
   ApplicationField,
-  FieldTypeMapping,
   SupportedFieldTypes,
   UploadedFileMeta,
 } from '../types';
 
-interface State {
-  fieldTypeMapping: FieldTypeMapping;
-}
-
-interface Props extends State {
+interface Props {
```

Comment:
Should we rename interface to give it more meaningful name if we happen to use it in somewhere else?